core/src/main/scala/cats/syntax/bitraverse.scala

```diff
@@ -11,14 +11,38 @@ private[syntax] trait BitraverseSyntax1 {
     fgagb: F[G[A], G[B]]
   ): NestedBitraverseOps[F, G, A, B] =
     new NestedBitraverseOps[F, G, A, B](fgagb)
+
+  implicit final def catsSyntaxLeftNestedBitraverse[F[_, _]: Bitraverse, G[_], A, B](
+    fgab: F[G[A], B]
+  ): LeftNestedBitraverseOps[F, G, A, B] =
+    new LeftNestedBitraverseOps[F, G, A, B](fgab)
+
+  implicit final def catsSyntaxRightnestedBitraverse[F[_, _]: Bitraverse, G[_], A, B](
+    fagb: F[A, G[B]]
+  ): RightNestedBitraverseOps[F, G, A, B] =
+    new RightNestedBitraverseOps[F, G, A, B](fagb)
 }
 
 final class BitraverseOps[F[_, _], A, B](val fab: F[A, B]) extends AnyVal {
   def bitraverse[G[_]: Applicative, C, D](f: A => G[C], g: B => G[D])(implicit F: Bitraverse[F]): G[F[C, D]] =
     F.bitraverse(fab)(f, g)
+  def traverse[G[_], C](f: B => G[C])(implicit F: Bitraverse[F], G: Applicative[G]): G[F[A, C]] =
```

Comment:
I'm not sure about this one and `sequence`, we already have them defined on `Traverse` and I think that can cause complications. Maybe we should remove these or rename them to `rightTraverse` and `rightSequence`

Reply:
I wasn't sure about this, but I'll change it to `rightTraverse` and `rightSequence` for now. You don't normally have syntax in scope to give you `traverse` and `sequence`, so I do think they're still valuable to have.